app/models/supermarket.rb

```diff
@@ -1,3 +1,4 @@
 class Supermarket < ApplicationRecord
   has_many :items
+  has_many :customers, through: :items
```

Comment:
Great work setting up these additional associations to make grabbing the customers very easy! Mooching associations of the child object is a Rails strategy that will take you far ✈️